### ISSUE TITLE
Suggestion: Let Pageant list valid certificates first

### ISSUE THREAD

**Issue by robbyjo** [NoMoreFood/putty-cac#32]
Just wondering if you can make Pageant list valid certificates first. Perhaps sort by descending order of the end validity date of each certificate? This will be useful for smart card that has a lot of certificates. Alternatively, there could be an option to filter out expired certificates from the list. Thanks!

**Comment by NoMoreFood** [comment]
Yeah, I think we can add an option for expired certificates.  Stay tuned.

**Comment by NoMoreFood** [comment]
For testing:

https://github.com/NoMoreFood/putty-cac/raw/48dc768d16b69fcf44b623f324f5f577fe967bf1/binaries/puttycac-0.70u5-installer.msi
https://github.com/NoMoreFood/putty-cac/raw/48dc768d16b69fcf44b623f324f5f577fe967bf1/binaries/puttycac-64bit-0.70u5-installer.msi

**Comment by NoMoreFood** [comment]
@robbyjo Have a chance to test?
